.travis.yml

```diff
@@ -4,7 +4,7 @@ scala:
    - 2.12.10
 
 script:
-  - sbt clean coverage test coverageReport
+  - sbt clean test

```

Comment:
Up to now the code coverage is not being uploaded anywhere like codecov, the tool we use at WESO, as it  has not been configured for this repo. So doesn't look like a big problem to remove the code coverage report at this point. 
Notice however that enabling code coverage properly would be better than removing the directives.